src/Mooc/Courses/Domain/Course.php

```diff
@@ -12,12 +12,14 @@ final class Course extends AggregateRoot
     private CourseId       $id;
     private CourseName     $name;
     private CourseDuration $duration;
+    private CourseCreatedAt $createdAt;
 
     public function __construct(CourseId $id, CourseName $name, CourseDuration $duration)
     {
         $this->id       = $id;
         $this->name     = $name;
         $this->duration = $duration;
+        $this->createdAt = new CourseCreatedAt(\DateTimeImmutable::createFromMutable(new \DateTime('now')));
```

Comment:
align

Reply:
Also, move `new \DateTime('now')` to the `create`  method. If it's here, we won't be able to recreate it correctly on tests.
```php
public function __construct(CourseId $id, CourseName $name, CourseDuration $duration, DateTimeImmutable $createdAt)
```